### ISSUE TITLE
Question about per-user configs

### ISSUE THREAD

**Issue by FraglyG** [waylaidwanderer/node-chatgpt-api#143]
Sorry for opening multiple issues at once

I just want to make sure that it's completely fine to make new classes of the ChatGPTClient for each user who uses it, I want to customize the chatgpt label per user.. or are there better ways of doing this?

**Comment by FraglyG** [comment]
CLIENT.userLabel = INSERT_USER_NAME
^ the issue with this is if multiple users uses it at the same time

though I think a way of unique userlabels would be to make the user label something unique like "USER1"
and then just change it after text completion, but that might mess with 

I noticed when I just used a generic name, it would reply as if that name is the user who talks to it ( makes sense ) 
but if it's something like "user" then it uses that in the context that there's multiple users for example..
this is just an example of how it uses the label in context

**Comment by waylaidwanderer** [comment]
Correct, the best way to do it right now is to create a new client instance for every user. If you want to provide dynamic info for the promptPrefix, I guess you can even create a new client instance for every message?
